lib/type.ts

```diff
@@ -521,6 +521,14 @@ export interface IChapter {
    * Duration offset is sampleOffset / format.sampleRate
    */
   sampleOffset: number;
+  /**
+   * Time where chapter starts
+   */
+  start: number;
+  /**
+   * Chapter track timescale
```

Comment:
That is a bit vague

Reply:
Type descriptions are updated on latest commit.